src/components/dev-hub/card-badge.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { colorMap, size } from './theme';
+import { P5 } from './text';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+const CardBadge = styled('div')`
```

Comment:
Let's also add `letter-spacing: 1px` here as well since the bold pushes everything together.